### ISSUE TITLE
Font not appear and can't change

### ISSUE THREAD

**Issue by jacklesphere** [unlayer/angular-email-editor#76]
<img width="510" alt="Screenshot 2022-02-04 at 11 26 27 AM" src="https://user-images.githubusercontent.com/45421149/152467171-48d7d0a4-283b-4140-83a9-045af52bdc4e.png">

how to load the font list in angular??

**Comment by jacklesphere** [comment]
> Passing the below in options solved the problem for me.
> 
> `fonts: { showDefaultFonts: false }`
> 
> Detailed documentation - https://docs.unlayer.com/docs/font-management
> 
> I think the `showDefaultFonts` logic is written in reverse.
> 
> So, in my html file I have `<email-editor [options]="options" (loaded)="editorLoaded()"></email-editor>`.
> 
> In my component, I have `options = {fonts: { showDefaultFonts: false }}`.

great, this solve my problem, thank you...

**Comment by Mallesh-Nagothi** [comment]
Yes, this solved my problem. The font logic is implemented in reverse!

**Comment by hridsn99** [comment]
This solved my problem with the initial display, but it messed up the styling in preview. I added the fonts I was using as custom fonts to help with that. I wish the developers would look into this, it's a funny bug

```
options = {
    fonts: { 
      showDefaultFonts: false,
      customFonts: [
        {
          label: "Playfair Display",
          value: "'Playfair Display',serif",
          url: "https://fonts.googleapis.com/css?family=Playfair+Display:400,700",
        },
        {
          label: "Montserrat",
          value: "'Montserrat',sans-serif",
          url: "https://fonts.googleapis.com/css?family=Montserrat:400,700",
        },
      ],
    }
}
```
